parts/ch11.tex

```diff
@@ -175,25 +186,30 @@ \subsection{Определенность в обработке зрительн
 
 (6) Мой экземпляр «Вечорки» от 3 мая, в тот момент, когда я его купил (по сравнению с тем, каким он стал, полежав под плошкой с кошачьей едой).
 
-Здесь строчки со второй по пятую играют обе роли. Строчка 4 --- пример общего класс строчки 3, и строчка 5 --- пример строчки 4. Строчка 6 --- это особый случай примера данного класса: \emph{проявление} . Разные состояния одного и того же предмета в разные моменты его жизни --- это его проявления. Интересно было бы узнать, понимают ли коровы, что веселый фермер, скармливающий им каждое утро щедрую порцию сена --- это один и тот же индивидуум, несмотря на его разные проявления?
+Здесь строчки со второй по пятую играют обе роли. Строчка 4 --- пример общего класс строчки 3, и строчка 5 --- пример строчки 4. Строчка 6 --- это особый случай примера данного класса: \emph{проявление}. Разные состояния одного и того же предмета в разные моменты его жизни --- это его проявления. Интересно было бы узнать, понимают ли коровы, что веселый фермер, скармливающий им каждое утро щедрую порцию сена --- это один и тот же индивидуум, несмотря на его разные проявления?
 
-Приведенный список кажется списком иерархии общности --- на вершине находится весьма широкая концептуальная категория, а внизу --- скромная конкретная вещь, локализованная во времени и пространстве. Однако идея о том, что «класс» должен обязательно быть очень широким и абстрактным, слишком ограничена. Дело в том, что наше мышление пользуется хитроумным принципом, который можно назвать \emph{принципом прототипа} :
-
-\emph{Любой частный случай может служить примером некого класса случаев} .
+Приведенный список кажется списком иерархии общности --- на вершине находится весьма широкая концептуальная категория, а внизу --- скромная конкретная вещь, локализованная во времени и пространстве. Однако идея о том, что «класс» должен обязательно быть очень широким и абстрактным, слишком ограничена. Дело в том, что наше мышление пользуется хитроумным принципом, который можно назвать \emph{принципом прототипа}:
+%TODO Сделать левую и правую границу центрирования меньше. \leftskip \rightskip не работает как надо, начинает расстягивать по всей строке
```

Comment:
Use \adjustwith{\parindent}{\parindent} in order to create a block of text with linewidth reduced by \parindent from both sides. However, I'm not sure it is necessary at all in this situation. What's bad in just a \begin{center}?